jmeter/0.1.0/README.md

```diff
@@ -4,36 +4,47 @@ This integration shows you how to run JMeter as a Kubernetes job using the [Jo
 
 **Note: This integration still has some restrictions! See the [restrictions section](#restrictions) for more information.**
 
-## Prerequisites 
+## Installation
 
 ### Step 1: Install the Job-Executor in your cluster
 
 Please follow the [Job-Executor documentation](https://github.com/keptn-contrib/job-executor-service) for the installation process.
 
 ### Step 2: Disable the JMeter-Service
 
-Disable normal jmeter-service (in case you don't want to run it in parallel):
+In case you have installed Keptns jmeter-service (e.g., `keptn install --use-case continuous-delivery` or `helm install jmeter-service https://github.com/keptn/keptn/releases/download/<KEPTN_VERSION>/jmeter-service-<KEPTN_VERSION>.tgz -n keptn --create-namespace --wait`), you can either temporarily disable or uninstall jmeter-service:
 
-```
+**Temporary disable keptn/jmeter-service:**
+
+```bash
 kubectl scale deployment/jmeter-service -n "keptn" --replicas=0
 ```
 
+Use
+```bash
+kubectl scale deployment/jmeter-service -n "keptn" --replicas=1
+```
+to re-enable it.
+
+**Uninstall keptn/jmeter-service:**
+
+```bash
+helm uninstall jmeter-service -n keptn
+```
+
+
 ### Step 3: Build your own JMeter Docker image
 
-Since JMeter has no official Docker image you will have to build your own one. Here is a Dockerfile with a basic JMeter installation.
+Since there are no official JMeter Docker images, we recommend to build your own (and customize it). Here is a Dockerfile with a basic JMeter installation, feel free to adapt!
 
 ```docker
 FROM alpine:3.15
 ENV env=production
-ARG JMETER_VERSION="5.1.1"
+ARG JMETER_VERSION="5.4.2"
 ENV JMETER_HOME /opt/apache-jmeter-${JMETER_VERSION}
 ENV	JMETER_BIN	${JMETER_HOME}/bin
 ENV	JMETER_DOWNLOAD_URL  https://archive.apache.org/dist/jmeter/binaries/apache-jmeter-${JMETER_VERSION}.tgz
 
-# Load additional extensions
```

Comment:
Fine with me if this is not used in the example as it looks like

Reply:
It's part of keptn/jmeter-service, but it's not needed in the "generic" example we are providing here.